assets/js/components/optin.js

```diff
@@ -111,8 +112,15 @@ class OptIn extends Component {
 					checked={ optIn }
 					onChange={ this.handleOptIn }
 				>
-					{ __( 'Help us improve the Site Kit plugin by allowing tracking of anonymous usage stats. All data are treated in accordance with ', 'google-site-kit' ) }
-					<a href="https://policies.google.com/privacy" target="_blank" rel="noopener noreferrer">{ __( 'Google Privacy Policy', 'google-site-kit' ) }</a>.
```

Comment:
This is incorrect because strings can't be properly translated when concatenated.